.buildkite/scripts/generate_xpack_packetbeat_pipeline.sh

```diff
@@ -31,20 +33,27 @@ steps:
           provider: "gcp"
           image: "${IMAGE_UBUNTU_X86_64}"
           machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
-        artifact_paths: "${BEATS_PROJECT_NAME}/build/*.xml"
+        artifact_paths:
+          - "$BEATS_PROJECT_NAME/build/*.xml"
+          - "$BEATS_PROJECT_NAME/build/*.json"
 
       - label: ":rhel: RHEL-9 Unit Tests"
         key: "mandatory-rhel9-unit-test"
-        command: ".buildkite/scripts/unit_tests.sh"
+        command: "cd $BEATS_PROJECT_NAME && mage build unitTest"
         agents:
           provider: "gcp"
           image: "${IMAGE_RHEL9_X86_64}"
           machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
-        artifact_paths: "${BEATS_PROJECT_NAME}/build/*.*"
+        artifact_paths:
+          - "$BEATS_PROJECT_NAME/build/*.xml"
+          - "$BEATS_PROJECT_NAME/build/*.json"
 
 
       - label: ":windows: Windows Unit Tests - {{matrix.image}}"
-        command: ".buildkite/scripts/win_unit_tests.ps1"
+        command:
+          - "Set-Location -Path $BEATS_PROJECT_NAME"
+          - "New-Item -ItemType Directory -Force -Path 'build'"
```

Comment:
we might have missed something, this should be similar to https://github.com/elastic/beats/blob/main/x-pack/packetbeat/Jenkinsfile.yml#L91

```suggestion
		command: |
          Set-Location -Path $BEATS_PROJECT_NAME
          mage build unitTest
```

Reply:
While we should be doing build and unitTests, the build was run successfully, so I guess its ok?
https://buildkite.com/elastic/beats-xpack-packetbeat/builds/1841#018ea97a-af2f-44af-8107-cda3dc56de05

Reply:
I think so